README.md

```diff
@@ -315,6 +321,40 @@ const Main = () => {
 }
 ```
 
+### Internalization (i18n)
+
+Ory Elements uses [react-intl](https://formatjs.io/docs/react-intl/) to format
+messages and provide translations. The default language is american English, but
+you can provide your own translations by using the `IntlProvider` component. The
+default translations of Ory Elements are located in the `src/locales` directory.
+They can be loaded using the `IntlProvider` from Ory Elements. Please note that
+it is necessary to wrap all Ory Element components either in the `IntlProvider`
+from `react-intl` or Ory Elements.
+
+```tsx
+import { IntlProvider } from "@ory/elements-react"
+
+const Main = () => {
+  return (
+    <IntlProvider locale="de">
```

Comment:
how can I add my own custom language as a consumer of the library? is there an easy override i can do or a file I can give it?